docs-src/cloud/api-keys/faq.md

```diff
@@ -0,0 +1,24 @@
+---
+id: api-keys-faq
+title: Frequently asked questions
+sidebar_label: FAQs
+description: FAQs API Keys.
+tags:
+  - explanation
+---
+
+**Q: Can I have multiple API Keys?**
+
+A: Yes, you can generate multiple keys to use in different services or for team members.
+
+**Q: How many API Keys can I have?**
+
+A: 10 per User.
+
+**Q: Is there an expiration for the API Keys?**
+
+A: Yes. During Preview, API Keys have an upper bound of 90 days. In a subsequent release, long-lived API Keys will become available. Temporal recommends rotating API Keys periodically.
+
+**Q: What should I do if I lost my API Key secret?**
+
+A: For security reasons, we only display the full key once upon creation. If you lose it, you'll need to generate a new one.
```

Comment:
May we append a Feedback section to the API Keys docs please, like exists for the Cloud Ops API docs? 

### Provide feedback

Your input is valuable.
While the Temporal Cloud Ops API is in Public Preview, we welcome your feedback.

You can provide feedback through the following channels:

- Submit request or feedback through a ZenDesk [ticket](/cloud/support#support-ticket)
- Open an issue in the [GitHub Repo](https://github.com/temporalio/cloud-ops-api)